<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="60270" started-at="2020-08-10T19:21:55Z" finished-at="2020-08-10T19:22:55Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="60270" started-at="2020-08-10T19:21:55Z" finished-at="2020-08-10T19:22:55Z">
      <class name="Practice.AppiumFramework.AppiumSecondProject">
        <test-method status="PASS" signature="SecondTest()[pri:0, instance:Practice.AppiumFramework.AppiumSecondProject@4667ae56]" name="SecondTest" duration-ms="23262" started-at="2020-08-11T00:52:31Z" finished-at="2020-08-11T00:52:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SecondTest -->
      </class> <!-- Practice.AppiumFramework.AppiumSecondProject -->
      <class name="Practice.AppiumFramework.AppiumFirstProject">
        <test-method status="PASS" signature="startEmulator()[pri:0, instance:Practice.AppiumFramework.AppiumFirstProject@1a968a59]" name="startEmulator" is-config="true" duration-ms="78" started-at="2020-08-11T00:51:54Z" finished-at="2020-08-11T00:51:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startEmulator -->
        <test-method status="PASS" signature="startTest()[pri:0, instance:Practice.AppiumFramework.AppiumFirstProject@1a968a59]" name="startTest" is-config="true" duration-ms="116" started-at="2020-08-11T00:51:55Z" finished-at="2020-08-11T00:51:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTest -->
        <test-method status="FAIL" signature="FirstTestCase()[pri:0, instance:Practice.AppiumFramework.AppiumFirstProject@1a968a59]" name="FirstTestCase" duration-ms="35830" started-at="2020-08-11T00:51:55Z" finished-at="2020-08-11T00:52:30Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Practice.AppiumFramework.AppiumFirstProject.FirstTestCase(AppiumFirstProject.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FirstTestCase -->
        <test-method status="FAIL" signature="reportsflush()[pri:0, instance:Practice.AppiumFramework.AppiumFirstProject@1a968a59]" name="reportsflush" is-config="true" duration-ms="0" started-at="2020-08-11T00:52:55Z" finished-at="2020-08-11T00:52:55Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Practice.AppiumFramework.AppiumFirstProject.reportsflush(AppiumFirstProject.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.afterRun(TestRunner.java:1023)
	at org.testng.TestRunner.run(TestRunner.java:628)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.run(TestNG.java:1024)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reportsflush -->
      </class> <!-- Practice.AppiumFramework.AppiumFirstProject -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
